shell/server/core.js

```diff
@@ -32,13 +32,17 @@ class SandstormCoreImpl {
     const _this = this;
     return inMeteor(() => {
       const hashedSturdyRef = hashSturdyRef(sturdyRef);
-      const token = ApiTokens.findOne(hashedSturdyRef);
-      const requirements = [
-        {
-          tokenValid: hashedSturdyRef,
```

Comment:
Are we sure we want to drop this requirement? I think it may be reasonable to do so, but I'm worried about accountability (ie. giving users the ability to revoke a capability that's been granted, even if it's been re-saved by another owner).